winterfell/controllers/claimController.js

```diff
@@ -7,10 +7,16 @@ var User = require('../models/user');
 var Form = require('../models/form');
 var UserValues = require('../models/userValues');
 var auth = require('../middlewares/auth');
+var MailingService = require('./../services/mailingService');
+var DocumentRenderingService = require('./../services/documentRenderingService');
 var _ = require('lodash');
+var Q = require('q');
```

Comment:
Agreed. There are linters out there which do this automatically, we should pick one and make it into a precommit hook eventually.